models/models.go

```diff
@@ -14,7 +14,7 @@ type DBModel struct {
 
 // Address represents one address
 type Address struct {
-	DBModel
+	//DBModel
```

Comment:
Every structure requires DBModel unless we can use Enumerations because it will help us in long run about the time of creation, etc and ID in debugging. So, no need to comment it out